pkg/ui/src/views/reports/containers/debug/index.tsx

```diff
@@ -79,285 +80,399 @@ function DebugPanelLink(props: { name: string, url: string,  note: string }) {
   );
 }
 
-export default function Debug() {
-  return (
-    <div className="section">
-      <Helmet title="Debug" />
-      <h1 className="base-heading page-title">Advanced Debugging</h1>
-      <div className="debug-header">
-        <InfoBox>
-          <p>
-            The following pages are meant for advanced monitoring and troubleshooting.
-            Note that these pages are experimental and undocumented. If you find an issue,
-            let us know through{" "}
-            <a href={ COMMUNITY_URL }>these channels.</a>
-          </p>
-        </InfoBox>
+interface IDebugProps {
+  nodesSummary: NodesSummary;
+  dataSource: DecommissionedNodeStatusRow[];
+}
+
+interface IDebugState {
+  selected: string;
+  location: string;
+  opened: boolean;
+  width: number;
+}
+
+const pattern = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@%_\+.~#?&=]*)?/;
```

Comment:
Can we do this without regexes please? I can't read them. If you're going to use one, please name it after what it matches and explain why it's necessary.